core/scss/helpers/_functions.scss

```diff
@@ -15,15 +17,15 @@
 }
 
 @function tint-color($color, $level) {
-  @return mix(white, $color, $level);
+  @return color.mix(white, $color, $level);
```

Comment:
Here and below, `$method: rgb` should be added so that [a hex string is generated](https://sass-lang.com/documentation/modules/color/#mix) rather than decimal values for an rgb function (rgb functions are not supported by bgcolor - see https://github.com/bootstrap-email/bootstrap-email/issues/280)

Note: This functionality is broken on the main branch as well, so it could be fixed outside / before this PR is merged.